autometrics/src/exemplars/tracing_opentelemetry.rs

```diff
@@ -2,11 +2,19 @@ use super::TraceLabels;
 use opentelemetry_api::trace::TraceContextExt;
 use std::iter::FromIterator;
 use tracing::Span;
-use tracing_opentelemetry::OpenTelemetrySpanExt;
+
+#[cfg(all(
+    feature = "exemplars-tracing-opentelemetry-0_20",
+    feature = "exemplars-tracing-opentelemetry-0_21"
+))]
```

Comment:
It currently cannot compile during doc generation. This would fix it: 

```suggestion
#[cfg(all(
    not(doc),
    all(
        feature = "exemplars-tracing-opentelemetry-0_20",
        feature = "exemplars-tracing-opentelemetry-0_21"
    )
))]
```

Reply:
Oh sweet, I didn't think of doing it this was. Thank you. Done in https://github.com/autometrics-dev/autometrics-rs/compare/a3443573979762d62196f60c422daccf6333e81c..e5f0cf42fdf9afcc1a6fd204bc5b60c3582716a5.